### DISCUSSION TITLE
Goodbye Microsoft Teams bar... Hello Productivity | xylem | Gordon Beeming

### DISCUSSION THREAD

**Discussion by giscus** [GordonBeeming/xylem#10] [Posts]
# Goodbye Microsoft Teams bar... Hello Productivity - Gordon Beeming

We've all been there, you're in a meeting, you start sharing your screen and things are going great... next up you need to do something at the top of your screen and you can't because the Microsoft Teams bar is in the way. You can't move it, you can't hide it, you can't do anything about it.

Or can you?

https://preview.gordonbeeming.com/blog/2024-05-16/goodbye-microsoft-teams-bar-hello-productivity

**Comment by piers-sinclair** [comment]
You're a life saver Gordon. This problem has been bugging me for what feels like years!

**Comment by JeanThirion** [comment]
You are an absolute legend. This is REALLY a good tip <3

**Comment by babakamyljanovssw** [comment]
Thanks Gordon for this tip, I had this issue many times where I couldn't open new browser tabs.
